### ISSUE TITLE
Can't start v0.12.3b16

### ISSUE THREAD

**Issue by qcol** [ActivityWatch/activitywatch#1033]
I have still problems with CPU 100%, (#917) so tried to run newest beta v0.12.3b16

Ubuntu 22.04.3 LTS

The server is working, it starts up http://localhost:5600/ but unfortunately nothing registers. There is also no icon in the tray. 

logs:
AW-QT
```
2024-02-22 07:22:28 [INFO ]: Started aw-qt...  (aw_qt.main:53)
2024-02-22 07:22:28 [WARNING]: Found matching file but was not executable: /home/qcol/activitywatch/aw-server-rust/aw-server.service  (aw_qt.manager:64)
2024-02-22 07:22:28 [WARNING]: Found matching file but was not executable: /home/qcol/activitywatch/aw-qt.desktop  (aw_qt.manager:64)
2024-02-22 07:22:28 [INFO ]: Found 7 bundled modules  (aw_qt.manager:85)
2024-02-22 07:22:28 [INFO ]: Found 0 system modules  (aw_qt.manager:119)
2024-02-22 07:22:28 [INFO ]: Starting module aw-server-rust  (aw_qt.manager:148)
2024-02-22 07:22:28 [INFO ]: Starting module aw-watcher-afk  (aw_qt.manager:148)
2024-02-22 07:22:28 [INFO ]: Starting module aw-watcher-window  (aw_qt.manager:148)
2024-02-22 07:22:30 [INFO ]: Creating trayicon...  (aw_qt.trayicon:206)
```

SERVER-RUST

```
[2024-02-22 07:22:28][[32mINFO[0m][aw_server]: Using DB at path "/home/qcol/.local/share/activitywatch/aw-server-rust/sqlite.db"
[2024-02-22 07:22:28][[32mINFO[0m][aw_server]: Using aw-webui assets at path None
[2024-02-22 07:22:28][[32mINFO[0m][aw_server::endpoints]: Starting aw-server-rust at 127.0.0.1:5600
[2024-02-22 07:22:28][[33mWARN[0m][rocket::launch]: Rocket has launched from http://127.0.0.1:5600
[2024-02-22 07:23:04][[31mERROR[0m][rocket::server::_]: No matching routes for GET /manifest.json.
[2024-02-22 07:23:04][[33mWARN[0m][rocket::server::_]: No 404 catcher registered. Using Rocket default.
[2024-02-22 07:23:10][[31mERROR[0m][rocket::server::_]: No matching routes for GET /manifest.json.
[2024-02-22 07:23:10][[33mWARN[0m][rocket::server::_]: No 404 catcher registered. Using Rocket default.
[2024-02-22 07:23:23][[33mWARN[0m][rocket::server::_]: Remote left: client disconnect before response started.
[2024-02-22 07:28:17][[33mWARN[0m][rocket::server::_]: Remote left: client disconnect before response started.
[2024-02-22 07:28:22][[32mINFO[0m][aw_datastore::datastore]: Created bucket aw-stopwatch
[2024-02-22 07:28:34][[33mWARN[0m][rocket::server::_]: Remote left: client disconnect before response started.
```

WATCHER-AFK
```
2024-02-22 07:22:31 [INFO ]: aw-watcher-afk started  (aw_watcher_afk.afk:62)
2024-02-22 07:22:32 [INFO ]: afkwatcher stopped because parent process died  (aw_watcher_afk.afk:82)
2024-02-22 07:22:36 [INFO ]: Connection to aw-server established by aw-watcher-afk  (aw_client.client:447)
```

WATCHER WINDOW

```2024-02-22 07:22:31 [INFO ]: aw-watcher-window started  (aw_watcher_window.main:62)
2024-02-22 07:22:32 [INFO ]: window-watcher stopped because parent process died  (aw_watcher_window.main:101)
2024-02-22 07:22:36 [INFO ]: Connection to aw-server established by aw-watcher-window  (aw_client.client:447)
```

**Comment by brayo-pip** [comment]
@qcol sorry for the late reply. Try using running `chmod 755 /home/qcol/activitywatch/aw-server-rust/aw-server.service` and `chmod  /home/qcol/activitywatch/aw-qt.desktop`

**Comment by qcol** [comment]
thank you ... unfortunately it did not help 
I currently have the situation that on version 0.12.2 when trying to report a month I have a timeout 3000 (I used to have a similar problem but after switching to rust it was OK, now I have it even on rust). On the other hand in 0.12.3 the reports are OK, but it doesn't collect the data :)

**Comment by BelKed** [comment]
There is a problem with `v0.12.3b16` which I can confirm. Please use `v0.12.3b15` in the meantime :)

**Comment by ErikBjare** [comment]
Should be fixed in the latest betas *after* v0.12.3b16.
